elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -32,6 +58,23 @@ def render(element_html, data):
             # back to a standard type (otherwise, do nothing)
             var_data = pl.from_json(var_data)
 
+            # Get comment, if it exists
+            var_matlab_comment = ''
+            var_mathematica_comment = ''
+            var_python_comment = ''
+            if pl.has_attrib(child, 'comment'):
+                var_comment = pl.get_string_attrib(child, 'comment')
+                var_matlab_comment = '% {}'.format(var_comment)
+                var_mathematica_comment = '(* {} *)'.format(var_comment)
+                var_python_comment = '# {}'.format(var_comment)
```

Comment:
To just be explicit about f-strings, they will look like this:
```
var_python_comment = f’# {var_comment}’
```

See 
https://docs.python.org/3/reference/lexical_analysis.html#f-strings

Also: 
https://realpython.com/python-f-strings/

Reply:
My Google search was very bad. Fixed now.